surefire-its/src/test/java/org/apache/maven/surefire/its/JUnitPlatformEnginesIT.java

```diff
@@ -93,12 +93,10 @@ public void platform() throws VerificationException
         String testClasspath = "[DEBUG] test(compact) classpath:"
                 + "  test-classes"
                 + "  classes"
-                + "  junit-jupiter-engine-" + jupiter + ".jar"
+                + "  junit-jupiter-api-" + jupiter + ".jar"
```

Comment:
Perfect! This makes sense for users. They will see only API.
But this reminds me to think about IDE which is not able to run the tests without implementation of engines. Did you try the same test in Eclipse or IntelliJ IDEA?

Reply:
At least _IntelliJ IDEA 2018.3 EAP (Community Edition)_ is able to add the matching engines at runtime. Not sure about Eclipse, though.

Reply:
Excellent, we should document a new user experience with examples:
1. with engine dependency
2. with API dependency (since of `3.0.0-M2`)
Do we have third option we can document with this fix?

Reply:
> Do we have third option we can document with this fix?

Don't think so. Will update the documentation when you approved this PR.